### DISCUSSION TITLE
Cannot install Phoenix

### DISCUSSION THREAD

**Discussion by adlodhi** [Arize-ai/phoenix#2719] [Q&A] [answered]
pip install llama-index-callbacks-arize-phoenix>1.3.0
 
gives the following error:

ERROR: Failed building wheel for hdbscan
ERROR: Could not build wheels for hdbscan, which is required to install pyproject.toml-based projects

OS: Windows 11

Please help

**Comment by mikeldking** [comment, marked as answer]
@adlodhi yes there's difficulty on windows with those packages. We recommend installing from conda since the packages are pre-compiled.
